include/libhal/percentage.hpp

```diff
@@ -21,6 +23,28 @@ class percentage
   constexpr static float_t min = -1.0;
   constexpr static float_t zero = 0.0;
 
+  constexpr static bool nearly_equal(percentage p_percentage1,
+                                     percentage p_percentage2,
+                                     float_t p_epsilon)
```

Comment:
```suggestion
                                     float_t p_epsilon = 1e-9)
```
Can you give this a default epsilon as defined in the example code we are referencing? I think 1e-9 or 1e-6 is good enough.